test/DocSearch-test.js

```diff
@@ -136,7 +166,7 @@ describe('DocSearch', () => {
       let actual = new DocSearch(options);
 
       // Then
-      expect(actual.algoliaOptions).toEqual('algoliaOptions');
+      expect(actual.algoliaOptions).toEqual({hitsPerPage: 5});
```

Comment:
The whole PR LGTM except for this part. This test seems unrelated. I would rather set an option of `{hitsPerPage: 10}` and test that.

We want to test that it accepts the options we pass, while this test assert that we keep the default value.